server/issuers.go

```diff
@@ -83,6 +83,7 @@ func (c *Server) GetLatestIssuerKafka(issuerType string, issuerCohort int16) (*I
 }
 
 // GetIssuers - get all issuers by issuer type
+// @TODO: This function is no longer needed. Delete.
```

Comment:
Is  this comment true for the V1 Issuers in grants?

Reply:
Fair enough. I'll change to recommend refactor or deprecation, as the functions are inefficient even we still need them now that there are not multiple issuers per cohort.